start_bs.py

```diff
@@ -132,10 +132,10 @@ def filter_camera_data(camera):
     zebra = Zebra('XF:19IDC-ES{Zeb:1}:', name='zebra')
     from nyxtools.vector import VectorProgram
     vector = VectorProgram("XF:19IDC-ES{Gon:1-Vec}", name="vector")
-    from nyxtools.pilatus import PilatusBase
-    detector = PilatusBase("XF:19IDC-ES{Det:Pil6M}", name="detector")
-    from nyxtools.flyer import NYXFlyer
-    flyer = NYXFlyer(vector, zebra, detector)
+    from nyxtools.pilatus import EigerBaseV26
+    detector = EigerBaseV26("XF:19IDC-ES{Det:Pil6M}", name="detector") # TODO: Update PV Base
+    from nyxtools.flyer_eiger2 import NYXEiger2Flyer
```

Comment:
I wouldn't use dashes in the filename if you can avoid it. you've translate it to "_" for the import, but it would be better if they are the same. just imagine grepping for "flyer-eiger2" and then having to also check "flyer_eiger2".

I think the import mechanism won't work with dashes anyways.

Reply:
I have renamed it to flyer_eiger2.py